modules/identity-provider-configuring-apache-request-header.adoc

```diff
@@ -1,100 +1,82 @@
 // Module included in the following assemblies:
 //
-// * orphaned
+// * authentication/identity_providers/configuring-request-header-identity-provider.adoc
 
 [id="identity-provider-configuring-apache-request-header_{context}"]
 = Configuring Apache authentication using request header
 
 This example configures an authentication proxy on the same host as the master.
 Having the proxy and master on the same host is merely a convenience and may not
 be suitable for your environment. For example, if you were already
-running a router
-on the master, port 443 would not be available.
+running a router on the master, port 443 would not be available.
 
 It is also important to note that while this reference configuration uses
 Apache's `mod_auth_gssapi`, it is by no means required and other proxies can
 easily be used if the following requirements are met:
 
-1. Block the `X-Remote-User` header from client requests to prevent spoofing.
-2. Enforce client certificate authentication in the `RequestHeaderIdentityProvider` configuration.
-3. Require the `X-Csrf-Token` header be set for all authentication request using the challenge flow.
-4. Only the `/oauth/authorize` endpoint and its subpaths should be proxied,
+. Block the `X-Remote-User` header from client requests to prevent spoofing.
+. Enforce client certificate authentication in the `RequestHeaderIdentityProvider` configuration.
+. Require the `X-Csrf-Token` header be set for all authentication requests using the challenge flow.
+. Only the `/oauth/authorize` endpoint and its subpaths should be proxied,
 and redirects should not be rewritten to allow the backend server to send the client to the correct
 location.
-5. The URL that proxies to `\https://<master>/oauth/authorize` must end with `/authorize` (with no trailing slash). For example:
-  * `\https://proxy.example.com/login-proxy/authorize?...` -> `\https://<master>/oauth/authorize?...`
-6. Subpaths of the URL that proxies to `\https://<master>/oauth/authorize` must proxy to subpaths of `\https://<master>/oauth/authorize`. For example:
-  * `\https://proxy.example.com/login-proxy/authorize/approve?...` -> `\https://<master>/oauth/authorize/approve?...`
-
-. Obtain the `mod_auth_gssapi` module from the
-link:https://access.redhat.com/solutions/392003[Optional channel].
-Install the following packages:
+. The URL that proxies to `\https://<namespace_route>/oauth/authorize` must end with `/authorize` (with no trailing slash). For example:
```

Comment:
using `\` now apaprently adds the backslash character even on the actual page, remove it where possible